packages/web-react/src/components/FileUploader/README.md

```diff
@@ -0,0 +1,386 @@
+
+```javascript
+<FileUploader>
+  <FileUploaderInput onError={(error) => console.error('My error log', error)} />
+  <FileUploaderList />
+</FileUploader>
+```
+
+### Uncontrolled
+
+```javascript
+<UncontrolledFileUploader
+  attachmentComponent={(props) => <FileUploaderAttachment key={props.id} {...props} />}
+  id="fileUploaderUncontrolled"
+  inputId="fileUploaderUncontrolledInput"
+  inputName="attachments"
+  inputProps={{
+    accept: '*',
+  }}
+  listId="fileUploaderUncontrolledList"
+  listProps={{
+    label: 'Attachments',
+  }}
+  onInputError={(error) => console.error('My error log', error)}
+/>
+```
+
+### Controlled Example with form submit
+
+```javascript
+const [submitting, setSubmitting] = useState < boolean > false;
+const [submitted, setSubmitted] = useState < boolean > false;
+const [validationText, setValidationText] = (useState < string) | (undefined > undefined);
+const [validationState, setValidationState] = (useState < ValidationState) | (undefined > undefined);
+
+const { fileQueue, addToQueue, clearQueue, onDismiss } = useFileQueue();
+
+const submitHandler = (event: FormEvent<HTMLFormElement>) => {
+  event.preventDefault();
+  const master = new Map(fileQueue);
+
+  setSubmitting(true);
+
+  // Callback after submit
+  setSubmitting(false);
+  setSubmitted(true);
+  clearQueue();
+};
+
+const errorHandler = (error: string) => {
+  setValidationState('error');
+  setValidationText(String(error));
+
+  setTimeout(() => {
+    setValidationState(undefined);
+    setValidationText(undefined);
+  }, 5000);
+};
+
+const resetStateHandler = () => {
+  setValidationState(undefined);
+  setValidationText(undefined);
+  setSubmitted(false);
+};
+
+<form onSubmit={submitHandler}>
+  <FileUploader
+    id="fileUploaderFormExample"
+    onDismiss={onDismiss}
+    fileQueue={fileQueue}
+    addToQueue={addToQueue}
+    clearQueue={clearQueue}
+    isDisabled={submitting}
+  >
+    <FileUploaderInput
+      id="fileUploaderFormExampleInput"
+      name="attachments"
+      label="Label"
+      linkText="Upload your file(s)"
+      labelText="or drag and drop here"
+      helperText="Max file size is 10 MB"
+      validationText={validationText}
+      validationState={validationState}
+      maxUploadedFiles={2}
+      onError={errorHandler}
+      isDisabled={submitting}
+      isMultiple
+    />
+    <FileUploaderList
+      id="fileUploaderFormExampleList"
+      label="Attachments"
+      inputName="attachments"
+      attachmentComponent={(props) => <FileUploaderAttachment key={props.id} {...props} />}
+    />
+  </FileUploader>
+  <div style={{ paddingTop: '1rem' }}>
+    <Button type="submit" color="primary" isDisabled={submitting || submitted}>
+      Submit form
+    </Button>
+    {submitted && (
+      <Button color="secondary" onClick={resetStateHandler} UNSAFE_style={{ marginLeft: '.5rem' }}>
+        Reset state
+      </Button>
+    )}
+  </div>
+</form>;
+```
+
+### Uncontrolled Example with form submit
```

Comment:
Please check all headings are Title Case (or Sentence case, but the same everywhere 🙂).

Reply:
done